src/main/java/adris/altoclef/tasks/resources/CollectSticksTask.java

```diff
@@ -0,0 +1,128 @@
+package adris.altoclef.tasks.resources;
+
+import adris.altoclef.AltoClef;
+import adris.altoclef.Debug;
+import adris.altoclef.TaskCatalogue;
+import adris.altoclef.tasks.CraftInInventoryTask;
+import adris.altoclef.tasks.ResourceTask;
+import adris.altoclef.tasks.construction.DestroyBlockTask;
+import adris.altoclef.tasks.movement.PickupDroppedItemTask;
+import adris.altoclef.tasks.movement.TimeoutWanderTask;
+import adris.altoclef.tasksystem.Task;
+import adris.altoclef.util.CraftingRecipe;
+import adris.altoclef.util.ItemTarget;
+import adris.altoclef.util.RecipeTarget;
+import adris.altoclef.util.helpers.ItemHelper;
+import net.minecraft.block.Block;
+import net.minecraft.block.Blocks;
+import net.minecraft.item.Item;
+import net.minecraft.item.Items;
+import net.minecraft.util.math.BlockPos;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+public class CollectSticksTask extends ResourceTask {
+
+    private final int _targetCount;
+
+    private static final Item[] _itemsToPickup;
+    static {
+        List<Item> itemsToPickup = new ArrayList<>(Arrays.asList(ItemHelper.LOG));
+        itemsToPickup.addAll(Arrays.asList(ItemHelper.PLANKS));
+        _itemsToPickup = itemsToPickup.toArray(new Item[itemsToPickup.size()]);
+    }
+
+    public CollectSticksTask(int targetCount) {
+        super(Items.STICK, targetCount);
+        _targetCount = targetCount;
+    }
+
+    @Override
+    protected boolean shouldAvoidPickingUp(AltoClef mod) {
+        return false;
+    }
+
+    @Override
+    protected void onResourceStart(AltoClef mod) {
+        mod.getBehaviour().push();
+        mod.getBlockTracker().trackBlock(Blocks.DEAD_BUSH);
+        mod.getBlockTracker().trackBlock(ItemHelper.itemsToBlocks(ItemHelper.LOG));
+        mod.getBlockTracker().trackBlock(ItemHelper.itemsToBlocks(ItemHelper.PLANKS));
```

Comment:
You don't need to worry about planks or logs because altoclef already knows how to get planks and logs. We really only have to worry about the bushes, so a lot of this is unecesarry.